extensions-contrib/opencensus-extensions/src/main/java/org/apache/druid/data/input/opencensus/protobuf/OpenCensusProtobufInputRowParser.java

```diff
@@ -85,6 +85,24 @@ public ParseSpec getParseSpec()
     return parseSpec;
   }
 
+  @JsonProperty
+  public String getMetricDimension()
```

Comment:
can we add a test for serialization / de-serialization? Here's an example on how to test those things https://github.com/confluentinc/druid/blob/6163335cd4a8ab47b8c06c7bc47c9eabbee27985/extensions-contrib/confluent-extensions/src/test/java/io/confluent/druid/transform/ExtractTransformTest.java#L138-L155

Reply:
Done.